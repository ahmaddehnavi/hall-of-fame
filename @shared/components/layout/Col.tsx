import {View} from 'react-native';

export const Col = View;