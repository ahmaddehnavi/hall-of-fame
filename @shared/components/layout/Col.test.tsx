import React from 'react';
import 'react-native';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import {Col} from './Col';

describe('Col Tests', () => {

    it('should match snapshot', () => {
        let tree = renderer.create(
            <Col

            />
        );
        expect(tree).toMatchSnapshot();
    });

});
