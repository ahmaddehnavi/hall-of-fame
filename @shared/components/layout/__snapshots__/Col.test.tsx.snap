// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col Tests should match snapshot 1`] = `<View />`;
